cms/static/sass/elements/_modal-window.scss

```diff
@@ -777,4 +780,44 @@
       }
     }
   }
+
+  .edit-unit-access, .edit-staff-lock {
+    .modal-section-content {
+    @include font-size(16);
+
+    .group-select-title {
+      font-weight: font-weight(semi-bold);
+      font-size: inherit;
+      margin-bottom: ($baseline/4);
+
+      .user-partition-select {
+        font-size: inherit;
+      }
+    }
+
+    .partition-group-directions {
+      padding-top: ($baseline/2);
+    }
+
+    .label {
+
+      &.deleted {
+        color: $red;
+      }
+
+      font-size: inherit;
+      margin-left: ($baseline/4);
```

Comment:
Anytime I see a margin-left (or right), I wonder what will happen with RTL. Have you tried it with RTL? There is an include for margin-left that will flip automatically for you-- @include margin-left()